.travis.yml

```diff
@@ -97,21 +96,11 @@ after_success:
       make INSTALL_ROOT="${PWD}/Pencil2D" install;
       rm -rf Pencil2D/usr/lib;
       echo "Creating AppImage...";
+      sed -i "/^Keywords\(\[[a-zA-Z_.@]\+\]\)\?=/d;/^Version=/cVersion=1.0" Pencil2D/usr/share/applications/pencil2d.desktop;
```

Comment:
Can you add a bit of documentation for what this line does please?

Reply:
It replaces the value of the version key with 1.0 and deletes all lines with a keyword key in any language.